blocks/transformations/text_statistics.py

```diff
@@ -1,17 +1,14 @@
-import nltk
-from nltk.corpus import wordnet as wn
-from typing import List, Any, Union, Tuple, Optional
+from typing import List, Tuple, Union
 from type import DataType
-from configs.constants import Const
 from collections import Counter
 from urlextract import URLExtract
-from .base import Augmenter
 from emoji import UNICODE_EMOJI_ENGLISH
 import re
 import numpy as np
+from .base import Transformation
 
 
-class StatisticAugmenter(Augmenter):
+class TextStatisticTransformation(Transformation):
```

Comment:
This is more suitable as a Transformation, as it doesn't preserve the original data (Augmenters do), it "replaces" it

Reply:
We should consider merging ``transformations`` and ``augmentors``. 
It seems like a fine line between the two: we could also pass the data back concatenated from the statisticaugmentor/transformator. Or are there any cases where the augmentor HAS to throw back the original data aswell (so its not an arbitrary decision?).

Reply:
I think it'll make my life a lot harder if we concatenated and returned the original data - we need to run different models for both input types.
I'm happy to merge transformations and augmentors. now or later?

Reply:
Later fine. It's just chore for now.